### PR TITLE
Additional machinecontroller fields in terraform config

### LINKED ISSUE #416: Update machine-controller to v1.1.8
Update machine-controller to v1.1.7

EDIT:
MC got expansion of GCE configs:
* https://github.com/kubermatic/machine-controller/pull/511
* https://github.com/kubermatic/machine-controller/pull/528
* https://github.com/kubermatic/machine-controller/pull/510 (this one is breaking, fix will be in 1.1.8)

### PR DESCRIPTION
**Pull request by kron4eg**
Fixes #416

```release-note
machinecontroller upgraded to 1.1.8
```

### CODE REVIEW

**Comment by kron4eg** [comment]
waiting for #453 to be merged to update TF configs and avoid huge conflict

**Comment by xmudrii** [comment]
The #453 is now merged. :shipit:

File: pkg/templates/machinecontroller/machines.go [hunk 1]
```diff
@@ -96,13 +96,9 @@ func createMachineDeployment(cluster *kubeoneapi.KubeOneCluster, workerset kubeo
...
 
 	return &clusterv1alpha1.MachineDeployment{
-		TypeMeta: metav1.TypeMeta{
-			APIVersion: "cluster.k8s.io/v1alpha1",
-			Kind:       "MachineDeployment",
-		},
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: metav1.NamespaceSystem,
-			Name:      fmt.Sprintf("%s-deployment", workerset.Name),
+			Name:      workerset.Name,
```
Comment (xmudrii) [inline pkg/templates/machinecontroller/machines.go]: What is going to happen when we upgrade the cluster? Will it recreate the worker nodes with the new name or keep using the old name?

Comment (kron4eg) [reply pkg/templates/machinecontroller/machines.go]: In existing clusters MachineDeployment wouldn't be changed.

Comment (xmudrii) [reply pkg/templates/machinecontroller/machines.go]: Sounds good.

**Pull request merged**